sql/catalyst/pom.xml

```diff
@@ -127,13 +127,4 @@
       </plugin>
     </plugins>
   </build>
-  <profiles>
-    <!-- Quasiquotes are merged into scala reflect from scala 2.11 onwards. -->
-    <profile>
```

Comment:
Nah, I think it's no longer needed. This was from the days where Spark SQL depended on the Quasiquote compiler for its code generation, but I think this profile became redundant once we started using Janino and removed the old codgen.